### PR TITLE
TELCODOCS-782 - adding Tech Preview label

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-782](https://issues.redhat.com//browse/TELCODOCS-782) - adding Tech Preview label

Version(s): 4.13

Issue:
https://issues.redhat.com/browse/TELCODOCS-782
part of https://issues.redhat.com/browse/CNF-5562

Link to docs preview: https://59841--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/enabling-workload-partitioning.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by slovern** [comment]
@eggfoobar @sunilcio This PR just adds the tech Preview text as requested in https://issues.redhat.com/browse/CNF-5562

Can you take a look and add the /lgtm if this is ok for you

File: scalability_and_performance/enabling-workload-partitioning.adoc [hunk 1]
```diff
@@ -1,11 +1,15 @@
 :_content-type: ASSEMBLY
 [id="enabling-workload-partitioning"]
+:FeatureName: Workload partitioning
 = Workload partitioning
+include::snippets/technology-preview.adoc[leveloffset=+0]
```
Comment (aireilly) [inline scalability_and_performance/enabling-workload-partitioning.adoc:5]:
```suggestion
:FeatureName: Workload partitioning
include::snippets/technology-preview.adoc[]
:!FeatureName: Workload partitioning
```

**Review by aireilly — COMMENTED** [review]
Looks good. Suggest doing the TP note as shown - this ensures that the `{FeatureName}` gets closed off after use.

**Review by snarayan-redhat — COMMENTED** [review]
LGTM! However, I see a comment from @aireilly which needs fixing.

**Comment by slovern** [comment]
Thanks for the review, @snarayan-redhat. And I've implemented the suggestion from @aireilly also

**Review by sheriff-rh — COMMENTED** [review]
LGTM! Merging.

File: scalability_and_performance/enabling-workload-partitioning.adoc [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 = Workload partitioning
+:FeatureName: Workload partitioning
+include::snippets/technology-preview.adoc[]
+:!FeatureName: Workload partitioning
 include::_attributes/common-attributes.adoc[]
 :context: enabling-workload-partitioning
 
 toc::[]
 
+:FeatureName: Workload partitioning
```
Comment (aireilly) [inline scalability_and_performance/enabling-workload-partitioning.adoc:12]: This should not have been added. This just resets the `FeatureName` variable and could lead to issues in other parts of the doc.

Comment (aireilly) [reply scalability_and_performance/enabling-workload-partitioning.adoc:12]: Fixed in https://github.com/openshift/openshift-docs/pull/59932

**Pull request merged**